guides/testing-rest-api-integrations-using-mockserver/testing-rest-api-integrations-using-mockserver.adoc

```diff
@@ -0,0 +1,221 @@
+common:header-top.adoc[]
+
+== Getting Started
+
+In this guide, you will learn how to
+
+* Create a Micronaut application that talks to an external REST API.
+* Test the external API integration using https://testcontainers.com/modules/wiremock/[Testcontainers WireMock module].
+
+
+common:requirements.adoc[]
+* https://www.testcontainers.org/supported_docker_environment/[A Docker environment supported by Testcontainers]
+
+common:completesolution.adoc[]
+
+== What we are going to achieve in this guide
+We will create a Micronaut application that talks to an external REST API.
+Then, we will test the external REST API integration using
+the https://testcontainers.com/modules/wiremock/[Testcontainers WireMock module].
```

Comment:
This guide explains how to test using both **WireMock** and **MockServer**. This sentence needs to be updated to reflect that.

Reply:
resolved via https://github.com/micronaut-projects/micronaut-guides/pull/1330/commits/c8926945aac1a1269f545be187bc65e114c5ef2e